hack/virtc/main.go

```diff
@@ -227,7 +228,14 @@ func buildPodConfig(context *cli.Context) (vc.PodConfig, error) {
 		Memory: vmMemory,
 	}
 
+	id := context.String("pod-id")
```

Comment:
I am fine with this change, but could you create the corresponding flag ? Indeed, there is no pod-id flag for the create command. And could you call it "id" instead of "pod-id" ? It is not needed to use pod-id for the naming because we are in the command pod ... thus it is pretty obvious.